backbone/src/main/java/org/researchstack/backbone/model/survey/NavigationStep.java

```diff
@@ -0,0 +1,24 @@
+package org.researchstack.backbone.model.survey;
+
+import org.researchstack.backbone.result.StepResult;
+import org.researchstack.backbone.result.TaskResult;
+import org.researchstack.backbone.step.QuestionStep;
+
+import java.util.List;
+
+/**
+ * Created by TheMDP on 12/31/16.
+ */
+
+public interface NavigationStep {
```

Comment:
Yes, the steps need multiple inheritance, since they usually extend some other step, and then also need to add NavigationStep support.